weaviate/collections/grpc/query.py

```diff
@@ -130,27 +131,45 @@ def __init__(
 
         self._generative: Optional[_Generative] = None
         self._rerank: Optional[Rerank] = None
-        self._sort: Optional[List[Sort]] = None
+        self._sort: Optional[List[_Sort]] = None
 
         self._group_by: Optional[_GroupBy] = None
 
         self._filters: Optional[_Filters] = None
 
-    def __parse_sort(self, sort: Optional[Union[Sort, List[Sort]]]) -> None:
+    def __parse_sort(
+        self, sort: Optional[Union[_Sort, List[_Sort], _Sorting, List[_Sorting]]]
```

Comment:
do we need List[_Sorting]? 

_Sort, List[_Sort] are BC and will be removed, right? and with _Sorting+chaining we have support for multiple sorts

Reply:
I think users will likely do `[Sort.by_property(), Sort.by_property()]` and provided compatibility for this. Do you think it better if the method just errors and points towards chaining instead?

Reply:
might just be me, but I think `Sort.by_property().by_property` has such a better UX than `[Sort.by_property(), Sort.by_property()]` that I don't see the point of providing lists - or is there a usecase that would be a lot easier with lists?

Reply:
I think the chaining is the preferred syntax with the `LIst[_Sorting]` there to help users if they go from the old `[Sort(), Sort()]` syntax to `Sort().by_().by_()`. I'll just add validation here, in the other query input validation PR, to catch this and throw for the user

Reply:
do you want to deprecate it with GA?